Add coverage for package count functionality (BZ1422552)

oshtaier opened the pull request:
```
nosetests tests/foreman/cli/test_product.py -m test_positive_package_count
.
----------------------------------------------------------------------
Ran 1 test in 155.500s

OK
```

ldjebran commented on tests/foreman/cli/test_product.py:
```diff
@@ -467,3 +469,31 @@ def test_positive_sync_by_label(self):
+        org = make_org()
+        for _ in range(3):
+            product = make_product({'organization-id': org['id']})
+            repo = make_repository({'product-id': product['id']})
```
for more clarity please add url=FAKE_1_YUM_REPO

ldjebran commented on tests/foreman/cli/test_product.py:
```diff
@@ -467,3 +469,31 @@ def test_positive_sync_by_label(self):
+                int(repo['content-counts']['packages']),
+                len(packages)
+            )
+            self.assertEqual(len(packages), 32)
```
please declare and use constants FAKE_1_YUM_REPO_PACKAGES_COUNT as we have already done with errata count

oshtaier commented:
@ldjebran addressed comments
```
nosetests tests/foreman/cli/test_product.py -m test_positive_package_count
.
----------------------------------------------------------------------
Ran 1 test in 194.511s

OK
```

ldjebran commented:
ACK
